behavior.nls

```diff
@@ -357,7 +357,11 @@ to move-around
   ; variable to make sure that the procedure find-defense
   ; is only run once
   let run-find-defense false 
-  ask agents with [not communicating][
+  ; at the beginning of the procedure no agent has moved yet
+  ask agents [
+    set moved false
+  ]
+  ask agents with [not communicating and not moved][
```

Comment:
We might get a problem here depending on when the agentset with [not communicating and not moved] is determined:
Iirc this actually happens when this function is called i.e. this would mean that every agent which is not communicating will execute it (as every agent will be "not moved" at this point in time). This assumption is central to the problem so lemme know if it is incorrect.
Assuming that above is correct it seems to me that it can happen that agents actually could move more than once in the movement step:

- First time when the prospective movement is triggered and then
- 2nd time b/c they're now on a defender child where they might move away with the normal movement probability, as there is no check which would prevent them from doing so.

I guess this is not intentional.